pkg/summary/summary.go

```diff
@@ -49,32 +49,60 @@ type Summary struct {
 	Namespaces  []string            `json:"namespaces"`
 }
 
+// Client checks if VPA objects should be created or deleted
```

Comment:
sounds good! ✨

Reply:
@rbren I got errors when trying to do this since Reconciler is created in the vpa package, I can't define new methods on it (with it?) in the summary package.

Reply:
Ah yes - you'd have to go back to the old function signatures, like `func Run(kubeClientVPA *kube.VPAClientInstance, vpaLabels map[string]string, excludeContainers string)`

Not a big deal though :)